### PR TITLE
Fix git url reference in package.json

### PR DESCRIPTION
**Pull request by mttrbrts**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ampretia/composer-wallet-ibmcos.git"
+    "url": "https://github.com/ampretia/composer-wallet-cloudant.git"
```
Comment (jt-nti) [inline package.json]: The repo just move organisation as well so this should be...
`"url": "https://github.com/IBM-Blockchain-Starter-Kit/composer-wallet-cloudant"`

**Comment by jt-nti** [comment]
@mttrbrts sorry to be a pain but could you include a DCO sign off in the commit please? Draft contribution guidelines on this wiki page if that helps https://github.com/IBM-Blockchain-Starter-Kit/IBM-Blockchain-Starter-Kit.github.io/wiki/Contributing-to-Blockchain-Starter-Kit

**Comment by mttrbrts** [comment]
@jt-nti All done. Thanks.

**Comment by jt-nti** [comment]
@mttrbrts commit message looks good, thanks. The repo url still refers to the old organisation though.

**Comment by jt-nti** [comment]
@mttrbrts in fact I think there are going to be a few ampretia hanging around so I'll fix those separately. Thanks

**Pull request merged**